lib/Sabberworm/CSS/CSSList/CSSList.php

```diff
@@ -142,6 +142,13 @@ private static function parseAtRule(ParserState $oParserState) {
 		} else {
 			//Unknown other at rule (font-face or such)
 			$sArgs = trim($oParserState->consumeUntil('{', false, true));
+			if (substr_count($sArgs, "(") != substr_count($sArgs, ")")) {
+				if($oParserState->getSettings()->bLenientParsing) {
```

Comment:
Do we need this `if`? Can’t we just throw an `UnsupportedTokenException` instead of a `SourceException` and the calling code will catch it in lenient mode? If that’s not the case, maybe it should be…

Reply:
Throwing an `UnsupportedTokenException` will not abort parsing, which leads to other kind of issues.

Reply:
Disregard this, then…